bundles/org.openhab.binding.comfoair/README.md

```diff
@@ -114,46 +112,57 @@ Available device options are shown in thing properties.
 | enthalpy#enthalpyLevel             | Number                   | Enthalpy Sensor Level           | Level of the enthalpy sensor                                                                | true      | yes      |
 | enthalpy#enthalpyTime              | Number                   | Enthalpy Sensor Timer           | Timer state of the enthalpy sensor                                                          | true      | yes      |
+| **Option Settings**                |                          |                                 |                                                                                             |           |          |
+| options#isPreheater                | Switch                   | Preheater                       | Preheater option installed                                                                  | true      | yes      |
+| options#isBypass                   | Switch                   | Bypass                          | Bypass option installed                                                                     | true      | yes      |
+| options#recuType                   | Number                   | ComfoAir Type                   | Type of the ComfoAir (1 = left / 2 = right)                                                 | true      | yes      |
+| options#recuSize                   | Number                   | ComfoAir Size                   | Size of the ComfoAir (1 = big / 2 = small)                                                  | true      | yes      |
+| options#isChimney                  | Switch                   | Chimney                         | Chimney option installed                                                                    | true      | yes      |
+| options#isCookerhood               | Switch                   | Cookerhood                      | Cookerhood option installed                                                                 | true      | yes      |
+| options#isHeater                   | Switch                   | Heater                          | Heater option installed                                                                     | true      | yes      |
+| options#isEnthalpy                 | Number                   | Enthalpy                        | Enthalpy option installed                                                                   | true      | yes      |
+| options#isEWT                      | Number                   | Geothermal Heat Exchanger       | Geothermal heat exchanger option installed                                                  | true      | yes      |
+| options#analogPriority             | Number                   | Analog Priority                 | Priority of analog inputs for highest fan level                                             | true      | yes      |
```

Comment:
Can these change over time? If they are fixed, they should be properties rather than Channels.

Reply:
Unlike my assumption, and thus the initial implementation as properties, it seems to be quite common that individual device options are swapped or deactivated during operation (as communicated by [users](https://community.openhab.org/t/comfoair-new-comfoair-binding/92891/44?u=boehan)).

Reply:
Ok, fair enough.